pkg/registry/controller/etcd/etcd.go

```diff
@@ -129,33 +129,33 @@ var _ = rest.Patcher(&ScaleREST{})
 
 // New creates a new Scale object
 func (r *ScaleREST) New() runtime.Object {
-	return &autoscaling.Scale{}
+	return &extensions.Scale{}
```

Comment:
Actually, please leave these as autoscaling.Scale(). The /apis/extensions/v1beta1/.../replicationcontrollers/.../scale path is available with v1beta1 scale subresource. HPA doesn't need to use the new path/resource yet.